dev/postgres/12/docker-compose.yml

```
@@ -0,0 +1,19 @@
+services:
+  postgres-12:
+    container_name: sequelize-pg-12
+    image: sushantdhiman/postgres:12
+    environment:
+      POSTGRES_USER: sequelize_test
+      POSTGRES_PASSWORD: sequelize_test
+      POSTGRES_DB: sequelize_test
+    ports:
+      - 23012:5432
```

let's keep the same port as before, it's what [the config](https://github.com/sequelize/sequelize/blob/main/test/config/config.ts#L67) expects

```suggestion
      - 23010:5432
```